simapp/params/weights.go

```diff
@@ -1,13 +0,0 @@
-package params
-
-// Default simulation operation weights for messages and gov proposals
-const (
-	DefaultWeightMsgSend                      int = 100
-	DefaultWeightMsgMultiSend                 int = 10
-	DefaultWeightMsgCreateValidator           int = 100
```

Comment:
btw, these same values are also in `testutil/sims/weights.go`. I think we can remove them from there too?